src/opcode.rs

```diff
@@ -1003,18 +1003,27 @@ opcode! {
         buf: { *const u8 },
         len: { u32 },
         ;;
-        flags: i32 = 0
+        flags: i32 = 0,
+
+        /// Set the destination address, for sending from an unconnected socket.
+        ///
+        /// When set, `dest_addr_len` must be set as well.
+        /// See also `man 3 io_uring_prep_send_set_addr`.
+        dest_addr: *const libc::sockaddr = core::ptr::null(),
+        dest_addr_len: libc::socklen_t = 0,
```

Comment:
Because of the generated setters, this generates two of them. This is consistent with `SendZc`, but I think it would be nicer (for both, although it would be breaking to change `SendZc` now) to have a single `dest_addr(addr: *const libc::sockaddr, addr_len: socklen_t)`, so it is impossible to forget to specify the length. Is such a thing possible?

Reply:
It's not too complicated so I think the status quo is good enough